### PR TITLE
Async-ify CODAPPluginHelper and Build project with Webpack

### PR DESCRIPTION
**Pull request by jsandoe**
* Various discrete changes to CodapPluginHelper to simplify and improve the maintainability of the code by employing async/await feature.

* Build the project with webpack to better manage page load and permit node_modules directory to remain transient.
  * Both the main Sonify plugin and the Info plugin are bundled.
  * Run "npm run build" to build.
  * Webpack complains the size of the built module is slightly too large. This is because of the size of NexusIO. There may be a way to trim this, but it is not explored, yet.
  * The build result is checked into Git. This is so the app will work on github.io.
  * It's probably possible to include csound and other javascript in the module, but this has
  not been done, yet.
  * Some small rearrangements of source. It evidently is possible to bundle other, non-js
  files, but this is not done at this time.

### CODE REVIEW

File: Sonify/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "private": true,
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "scripts": {
+    "build": "webpack --config webpack.config.js",
+    "test": "echo \"No automated tests at this time. See test/test-cases.md.\" && exit 1"
+  },
+  "author": "Takahiko Tsuchiya",
```
Comment (ttsuchiya) [inline Sonify/package.json:14]: Please feel free to change this to "Concord Sonification Team" or something...

File: Sonify/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "license": "ISC",
+  "dependencies": {
+    "nexusui": "^2.1.6",
```
Comment (ttsuchiya) [inline Sonify/package.json:17]: If I remember correctly, the NexusUI used in the plugin might be v1 (need to check). v1 -> v2 was a complete rewrite and had many breaking changes.

**Review by ttsuchiya — APPROVED** [review]
Thank you, Jonathan. The changes look great. This will now allow the use of advanced tools like React, TypeScript, etc. if ever needed. 

I suggest we also incorporate prettier so the code style (e.g., indentation) is standardized.

File: Sonify/package.json (hunk 2 above)
Comment (jsandoe) [reply Sonify/package.json:17]: I looked at NexusUI.js in the old code-base and could not find a version number. I haven't seen any problems with the version we are now using, so maybe the incompatibilities are in unused parts. The only problems I have seen is that this library contributes a good deal to the size of the uploaded module.

**Pull request merged**